flash_examples/custom_task.py

```diff
@@ -77,12 +84,13 @@ def from_dataset(cls, x: ND, y: ND, preprocess: Preprocess, batch_size: int = 64
 
         x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=.20, random_state=0)
 
-        dm = cls.from_load_data_inputs(
-            train_load_data_input=(x_train, y_train),
-            test_load_data_input=(x_test, y_test),
+        dm = cls.from_data_source(
+            "numpy",
```

Comment:
@ethanwharris now seeing this the example - I see a bit redundant to specify in the constructor the data source type and again here